web/src/Root.jsx

```diff
@@ -340,6 +362,7 @@ class Root extends PureComponent {
       connectionTerminated,
       errLoggingOut,
     } = this.state;
+    this.checkIsHelmManaged();
```

Comment:
If we only need to check this when the component first mounts (when the user first gets on the page) we can run this in componentDidMount. 
Otherwise, the app will continuously run checkIsHelmManaged. 

```
componentDidMount = async () => {
    this.onRootMounted();
    this.interval = setInterval(async () => await this.ping(), 10000);
    this.checkIsHelmManaged()
  };
```